src/components/BottomBarCourse.vue

```diff
@@ -134,69 +110,56 @@ export default {
 
   computed: {
     CUROverallRating() {
-      if (this.overallRating === 0) return '';
-      if (!this.overallRating) return 'No Reviews';
-      return this.overallRating;
+      if (this.courseObj.overallRating === 0) return '';
+      if (!this.courseObj.overallRating) return 'N/A';
+      return this.courseObj.overallRating;
     },
 
     CURDifficulty() {
-      if (this.difficulty === 0) return '';
-      if (!this.difficulty) return 'No Reviews';
-      return this.difficulty;
+      if (this.courseObj.difficulty === 0) return '';
+      if (!this.courseObj.difficulty) return 'N/A';
+      return this.courseObj.difficulty;
     },
 
     CURWorkload() {
-      if (this.workload === 0) return '';
-      if (!this.workload) return 'No Reviews';
-      return this.workload;
+      if (this.courseObj.workload === 0) return '';
+      if (!this.courseObj.workload) return 'N/A';
+      return this.courseObj.workload;
     }
   }
 };
 </script>
 
 <style scoped lang="scss">
-  .bottombar {
+  .bottombarcourse {
     position: fixed;
     bottom: 0;
     left: 0;
     width: 100%;
     background-color: #FFF;
 
+    left: 29.5rem;
+    width: calc(100vw - 29.5rem);
+
     &-tabs {
       height: 0px;
     }
 
-    &-title {
-      padding-left: 24px;
-      height: 40px;
-      line-height: 40px;
-      background: #25A2AA;
-      color: #FFF;
-      font-size: 16px;
-      cursor: pointer;
-      filter: brightness(100%);
-
-      &:hover {
-        filter: brightness(95%);
-        transition: all 0.2s ease;
-      }
-    }
-
     &-bar {
       width: 100%;
 
       &-info {
         float: left;
-        height: 261px;
-        width: 35%;
+        height: 16.31rem;
```

Comment:
Same as above. Should start with at least an even number pixel value (260 or 262) before dividing by 16. Thanks for switching to rem though :)

Reply:
Same as above and now it's 16.25rem